neo4j/assets/configuration/spec.yaml

```diff
@@ -1,32 +1,82 @@
 name: Neo4j
+
 files:
 - name: neo4j.yaml
+  
   options:
   - template: init_config
-    options:
-    - template: init_config/http
-    - template: init_config/default
+        
   - template: instances
     options:
-    - name: neo4j_url
+    - name: host
```

Comment:
Removing `neo4j_url` is a significant breaking change. Any user who upgrades will have errors and difficulty debugging. The change should be documented in detail in the README.md at minimum (specifically in Configuration section)